zypp-plugin/snapper_zypp_plugin.cc

```diff
@@ -0,0 +1,363 @@
+// getenv
+#include <stdlib.h>
+// getppid
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <boost/format.hpp>
```

Comment:
There is already sformat in storage/AppUtil.h. Is something else really needed (esp. something else that has not been included in C++20)?

Reply:
Thanks for the pointer, I will check that